okhttp-logging-interceptor/src/main/java/okhttp3/logging/HttpLoggingInterceptor.java

```diff
@@ -251,6 +259,27 @@ public Level getLevel() {
     return response;
   }
 
+  private boolean isPlainText(Buffer buffer) throws IOException {
+    Buffer out = new Buffer();
+    long bytes = buffer.size() < 64 ? buffer.size() : 64;
+    buffer.copyTo(out, 0, bytes);
```

Comment:
You can probably avoid creating an entire additional buffer, just use the original buffer and call `getByte(i)`